### DISCUSSION TITLE
伪类样式改如何修改

### DISCUSSION THREAD

**Discussion by guzily** [arco-design/arco-design-vue#2034] [Q&A]
我想修改image组件在预览时的伪元素，如图是我想要的效果，在浏览器调试工具中可以实现更改
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/74231693/211452892-06cb5df3-41a4-444e-9d2a-dba0c1ba798d.png">
但在vue代码中更改伪类元素，这是代码，这些样式不生效
`:deep(.arco-image-preview-img-container::before) {
    display: inline-block;
    width: 20px;
    height: 20px;
    vertical-align: top;
    content: 'xx';
    background: yellow;
  }`
请问该如何修改arco组件的伪类样式呢？

**Comment by kaikaibenkai** [comment]
去掉 `scoped` 和 `:deep()` 试试？
